app/web/features/search/SearchPage.tsx

```diff
@@ -52,193 +58,211 @@ const useStyles = makeStyles((theme) => ({
-      if (!e.defaultPrevented) {
-        setSelectedResult(undefined);
+        return service.search.userSearch(
+          {
+            query: queryName,
+            bbox: locationResult.bbox,
+            lastActive:
+              lastActiveComparation === 0 ? undefined : lastActiveFilter,
+            hostingStatusOptions:
+              hostingStatusFilterComparation === 0
+                ? undefined
+                : [hostingStatusFilter],
+            numGuests: numberOfGuestFilter,
+            completeProfile:
+              completeProfileFilter === false
+                ? undefined
+                : completeProfileFilter,
+          },
+          pageParam
+        );
+      },
+      {
+        getNextPageParam: (lastPage) =>
+          lastPage.nextPageToken ? lastPage.nextPageToken : undefined,
       }
-    };
-
-    //bind event handlers for map events (order matters!)
-    map.current.on(
-      "click",
-      layers.unclusteredPointLayer.id,
-      handleMapUserClick
     );
-    map.current.on("click", handleMapClickAway);
 
-    map.current.on("sourcedata", handleMapSourceData);
-
-    return () => {
-      if (!map.current) return;
+  // Relocate map everytime boundingbox changes
+  useEffect(() => {
+    map.current?.fitBounds(locationResult.bbox, {
+      maxZoom: selectedUserZoom,
+    });
+  }, [locationResult.bbox]);
 
-      //unbind event handlers for map events
-      map.current.off("sourcedata", handleMapSourceData);
-      map.current.off("click", handleMapClickAway);
-      map.current.off(
-        "click",
-        layers.unclusteredPointLayer.id,
-        handleMapUserClick
+  // Initial bounding box
+  useEffect(() => {
+    if (isLoadingUser === false && userData && bbox.join() === "0,0,0,0") {
+      map.current?.fitBounds(
+        [userData.lng, userData.lat, userData.lng, userData.lat],
+        {
+          maxZoom: 4,
+        }
       );
-    };
-  }, [handleMapUserClick, handleMapSourceData]);
 
-  const initializeMap = (newMap: MaplibreMap) => {
-    map.current = newMap;
-    newMap.on("load", () => {
-      addClusteredUsersToMap(newMap);
-    });
-  };
+      map.current?.fitBounds(map.current?.getBounds());
+    }
+  }, [isLoadingUser]);
+
+  let errorMessage = error?.message || undefined;
+  if (errorUser) {
+    errorMessage = `${errorMessage || ""} ${errorUser}`;
+  }
 
   return (
-    <>
+    <QueryClientProvider client={queryClient}>
       <HtmlMeta title={t("global:nav.map_search")} />
       <div className={classes.container}>
+        {/* Desktop */}
         <Hidden smDown>
           <SearchResultsList
-            handleResultClick={setSelectedResult}
-            handleMapUserClick={handleMapUserClick}
-            map={map}
-            updateMapBoundingBox={updateMapBoundingBox}
-            selectedResult={selectedResult?.userId}
-            searchFilters={searchFilters}
+            setIsFiltersOpen={setIsFiltersOpen}
```

Comment:
I was thinking about creating a context and put there all these variables, to avoid the long prop injection, anyhow, even if I don't implement the context, this can be improved, but I suggest to do it in the future to not make this PR unnecessary long